grant_access/example.txt

```diff
@@ -0,0 +1,3 @@
+# filename must match the target resource
+# file contents: feedstock name(s), sans -feedstock, one per line
+foo-feedstock
```

Comment:
```suggestion
foo
```

For consistency with the other parts of `admin-requests`, this should work _without_ the -feedstock suffix. Can you double check this is true? Thanks